airbyte-integrations/connectors/source-paypal-transaction/metadata.yaml

```diff
@@ -11,7 +11,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: d913b0f2-cc51-4e55-a44c-8ba1697b9239
-  dockerImageTag: 2.1.2
+  dockerImageTag: 2.1.3
```

Comment:
Nit: I would say this is a minor as it's new functionality

Reply:
good call. fixed!